setup.cfg

```diff
@@ -55,7 +55,7 @@ install_requires =
     tqdm
     click
     Jinja2
-    ax-platform
+    ax-platform==0.5.0
```

Comment:
Maybe just set to less than 1 instead. Also, how does this compare to the ax version in the top level requirements file as opposed to the docs requirements file?